react/features/filmstrip/components/native/Thumbnail.js

```diff
@@ -17,7 +17,7 @@ import { Container } from '../../../base/react';
 import { connect } from '../../../base/redux';
 import { StyleType } from '../../../base/styles';
 import { getTrackByMediaTypeAndParticipant } from '../../../base/tracks';
-
+import { DisplayNameLabel } from '../../../display-name/components/native';
```

Comment:
same here, I think the import should be done from the `display-name` instead of the `components/native`

Reply:
Yeah, same as above